content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+
+```rust
+impl SpecializedPipeline for MyPipeline {
+    type Key = MyPipelineKey;
+    fn specialize(&self, key: Self::Key) -> RenderPipelineDescriptor {
+        // define a specialized pipeline for the given key here
+    }
+}
+```
+
+Implementors of this trait can then easily and cheaply access specialized pipeline variants (with automatic per-key caching and hot-reloading). If this feels too abstract / advanced, don't worry! This is a "mid level power-user tool", not something most Bevy App developers need to contend with.
+
+### Simpler Shader Stack
+
+Bevy now uses [Naga](https://github.com/gfx-rs/naga) for all of its shader needs. As a result, we were able to remove all of our complicated non-rust shader dependencies: `glsl_to_spriv`, `shaderc`, and `spirv_reflect`. `glsl_to_spriv` was a major producer of platform-specific build dependencies and bugs, so this is a huge win!
+
+### Features Ported to the New Renderer
+
+Render logic for internal Bevy crates had to be rewritten in a number of cases to take advantage of the new renderer. The following people helped with this effort:
+
+* bevy_sprites: @cart, @StarArawn 
+* bevy_pbr: Rob Swain (@superdump), @aevyrie, @cart, @zicklag, @jakobhellermann
+* bevy_ui: @Davier
+* bevy_text: @Davier
+* bevy_gltf: Rob Swain (@superdump)
+
+### WebGL2 Support
+
+<div class="release-feature-authors">authors: @zicklag, @mrk-its, @mockersf, Rob Swain (@superdump)</div>
+
+Bevy now has built-in support for deploying to the web using WebGL2 / WASM, thanks to @zicklag adding a native WebGL2 backend to wgpu. There is now no need for the third party `bevy_webgl2` plugin. Any Bevy app can be deployed to the web by running the following commands:
+
+```rust
+cargo build --target wasm32-unknown-unknown
+wasm-bindgen --out-dir OUTPUT_DIR --target web TARGET_DIR
+```
+
+The New Bevy Renderer developers had to carefully operate within the limits of WebGL2 (ex: storage buffers and compute shaders aren't supported in WebGL2), but the results were worth it! You can try out Bevy's WASM support in your browser using our new [Bevy Examples](/examples) page.
+
+[![wasm bevy examples](bevy_examples_wasm.png)](/examples)
+
+### Infinite Reverse Z Perspective Projection
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+For improved precision in the "useful range", the industry has largely adopted "reverse projections" with an "infinite" far plane. The new Bevy renderer was adapted to use the "right handed infinite reverse z" projection. [This Nvidia article](https://developer.nvidia.com/content/depth-precision-visualized) does a great job of explaining why this is so worthwhile. 
+
+### Compute Shaders
```

Comment:
Also the "computer shader game of life" example's link should probably target the 0.6 git tag (when the release is baked) to avoid breaking when `main` changes in the future.

Reply:
Or latest, if we think that we're going to keep this path stable.

Reply:
I think it makes more sense for links in release notes to point to that specific release.